homeassistant/components/accuweather/sensor.py

```diff
@@ -65,7 +61,8 @@ class AccuWeatherSensorDescription(
     """Class describing AccuWeather sensor entities."""
 
     attr_fn: Callable[[dict[str, Any]], dict[str, StateType]] = lambda _: {}
-    unit_fn: Callable[[bool], str | None] = lambda _: None
+    metric_unit: str | None = None
+    us_customary_unit: str | None = None
```

Comment:
This was refactored to avoid multi-line lambdas such as
```python
        unit_fn=lambda metric: SPEED_KILOMETERS_PER_HOUR
        if metric
        else SPEED_MILES_PER_HOUR,
```

Reply:
Good idea